src/ensembl/src/shared/components/external-link/ExternalLink.tsx

```diff
@@ -38,7 +38,7 @@ const ExternalLink = (props: ExternalLinkProps) => {
   return (
     <span className={styles.container}>
       <LinkIcon className={iconClass} />
-      <a className={linkClass} href={props.to}>
+      <a className={linkClass} href={props.to} target="_blank" rel="noreferrer">
```

Comment:
Noreferrer is great! Perhaps `nofollow noreferrer` (to instruct Google crawler not to follow this link and thus not to go to the other site) would be even better?

Reply:
yup I can add that.